release_notes/46-release-notes.adoc

```diff
@@ -0,0 +1,320 @@
+* The endpoint `/v2/compliance/scan/configurations/reports/run` method has changed from `PUT` to `POST`.
+
+
+[id="documentation-updates-450_{context}"]
+== Documentation updates
+
+
+
+
+[id="deprecated-and-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in earlier releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+For the most recent list of major functionality deprecated and removed, see the following table.
+Additional information about some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+* GA: General Availability
+* TP: Technology Preview
+* DEP: Deprecated
+* REM: Removed
+* NA: Not applicable
+
+.Deprecated and removed features tracker
+[cols="4,1,1,1",options="header"]
+|===
+|Feature |{product-title-short} 4.4 |{product-title-short} 4.5 |{product-title-short} 4.6
+
+| API token authentication for Red Hat OpenShift Cluster Manager^[1]^
+| GA
+| GA
+| DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|
+
+| Google Container Registry integration^[2]^
+| GA
+| GA
+| DEP
+
+|Kernel support packages and driver download functionality ^[3]^
+|NA
+|DEP
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|DEP
+
+| `rhacs-collector*` and `rhacs-collector-slim*` images^[4]^
+|NA
+|DEP
+|DEP
+
+|`roxctl connectivity-map`
+|DEP
+|DEP
+|
+
+|`roxctl generate netpol`
+|DEP
+|DEP
+|
+
+|`stackrox-db` Central PVC^[5]
+|GA
+|GA
+|REM
+
+|`/v1/availableAuthProviders` endpoint
+|GA
+|DEP
+|DEP
+
+
+|`/v1/clustercves/suppress` APIs^[6,7]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs^[6,7]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/cve/requests` APIs^[8]^
+|DEP
+|DEP
+|REM
+
+|`/v1/nodecves/suppress` APIs^[6,7]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs^[6,7]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|NA
+|DEP
+|DEP
+
+|`/v1/tls-challenge` endpoint
+|DEP
+|DEP
+|
+
+|Vulnerability Management (1.0) menu item^[9]^
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|===
+
+. API token authentication has been deprecated by Red Hat OpenShift Cluster Manager. The corresponding cloud source integration now uses service accounts for authentication.
+. The Google Container Registry integration is now deprecated. Users should use Artifact Registry as a registry replacement and Scanner V4 as a scanner replacement.
```

Comment:
If you want a link - https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr